docs/rbac/example.tf

```diff
@@ -0,0 +1,44 @@
+# Create the Google Cloud terraform provider
+provider "google" {
+  project = "{{YOUR_PROJECT_ID}}"
+  region  = "us-central1"
+  zone    = "us-central1-c"
+}
+# Create a custom IAM role to bind to our GCP service account
+
+# Declare a special IAM role
+resource "google_project_iam_custom_role" "minimal-k8s-role" {
+  role_id     = "Minimalk8sIamRole"
+  title       = "Minimal IAM role for GKE access"
+  description = "Bare minimum permissions to access the kubernetes API for using the Jenkins GKE plugin."
+  project = "{{YOUR_PROJECT_ID}}"
```

Comment:
Nit: The `=` on this line doesn't match up

Reply:
Done